httpx/middleware/request_id.go

```diff
@@ -42,6 +42,13 @@ func (h *RequestID) ServeHTTPContext(ctx context.Context, w http.ResponseWriter,
 	return h.handler.ServeHTTPContext(ctx, w, r)
 }
 
+func AddRequestIDToCtx(h httpx.Handler) httpx.Handler {
```

Comment:
If you continue with this, this can be simplified to:

```go
func AddRequestIDToCtx(h httpx.Handler) httpx.Handler {
        return ExtractRequestId(h)
}
```